### ISSUE TITLE
Stations With Special Characters

### ISSUE THREAD

**Issue by keltonhalbert** [sharppy/SHARPpy#38]
Stations with special characters currently do not work for the model data. This is because the URLs on the data server use a different syntax (for example, # gets replaced with %23). It should be a relatively easy fix, but I'm opening an issue because it is something routinely forget about and should be fixed at some point.

**Comment by wblumberg** [comment]
@tsupinie has made some leeway on this bug with the new picker.  It's either that or he's made changes in one of his bug fixes for the HTML-based SHARPpy picker and fixed this issue.  I'm not certain where the fix may be.  Tim could probably elaborate on his changes for this.

**Comment by wblumberg** [comment]
It's not in the newest edition of the picker, but I remember Tim mentioning that fix was in his list.

**Comment by keltonhalbert** [comment]
Yeah, I had seen him talk about it on Central OK Wx. I figured I would do this to help maintain bug reporting standards as things start to take off and we get more suggestions/reports from other people.

**Comment by wblumberg** [comment]
Yeah, I agree.  With the combination of Facebook, email, ST, Twitter, the whole bugs/recommendations reporting will get out of hand if we don't control them correctly (strongly agreeing with your ST post).

**Comment by keltonhalbert** [comment]
I've implemented a fix for this issue that has been pushed to the map-selector branch. The bug fix will not go live until the map-selector branch is ready to be merged with the master branch, but I'm going to go ahead and close this issue since it has been resolved.

It turns out, URLLIB has a function for handling this sort of formatting. I just wrapped the URL in that function in all is well. This means that all data sites should now work.
